test/topology/test_mv.py

```diff
@@ -0,0 +1,118 @@
+    await cql.run_async("CREATE KEYSPACE " + keyspace + " " + opts)
+    try:
+        yield keyspace
+    finally:
+        await cql.run_async("DROP KEYSPACE " + keyspace)
+
+previously_used_table_names = []
+@asynccontextmanager
+async def new_test_table(cql, keyspace, schema, extra=""):
+    """
+    A utility function for creating a new temporary table with a given schema.
+    Because Scylla becomes slower when a huge number of uniquely-named tables
+    are created and deleted (see https://github.com/scylladb/scylla/issues/7620)
+    we keep here a list of previously used but now deleted table names, and
+    reuse one of these names when possible.
+    This function can be used in a "async with", as:
+       async with create_table(cql, test_keyspace, '...') as table:
+    """
+    global previously_used_table_names
+    if not previously_used_table_names:
+        previously_used_table_names.append(unique_name())
+    table_name = previously_used_table_names.pop()
+    table = keyspace + "." + table_name
+    await cql.run_async("CREATE TABLE " + table + "(" + schema + ")" + extra)
+    try:
+        yield table
+    finally:
+        await cql.run_async("DROP TABLE " + table)
+        previously_used_table_names.append(table_name)
+
+@asynccontextmanager
+async def new_materialized_view(cql, table, select, pk, where, extra=""):
+    """
+    A utility function for creating a new temporary materialized view in
+    an existing table.
+    """
+    keyspace = table.split('.')[0]
+    mv = keyspace + "." + unique_name()
+    await cql.run_async(f"CREATE MATERIALIZED VIEW {mv} AS SELECT {select} FROM {table} WHERE {where} PRIMARY KEY ({pk}) {extra}")
+    try:
+        yield mv
+    finally:
+        await cql.run_async(f"DROP MATERIALIZED VIEW {mv}")
+
+@pytest.mark.asyncio
+async def test_mv_tombstone_gc_setting(manager):
+    """
+    Test that the tombstone_gc parameter can be set on a materialized view,
+    during CREATE MATERIALIZED VIEW and later with ALTER MATERIALIZED VIEW.
+    Note that the test only tests the ability to set this parameter - not
+    its actual function after it's set.
+    The tombstone_gc feature is Scylla-only, so the test is scylla_only.
+
+    tombstone_gc=repair is not supported on a table with RF=1, and RF>1
+    is not supported on a single node, which is why this test needs to
+    be here and not in the single-node cql-pytest.
+    """
+    cql = manager.cql
+    async with new_test_keyspace(cql, "WITH REPLICATION = { 'class': 'NetworkTopologyStrategy', 'replication_factor': 3 }") as keyspace:
+        async with new_test_table(cql, keyspace, "p int primary key, x int") as table:
+            # Adding "WITH tombstone_gc = ..." In the CREATE MATERIALIZED VIEW:
+            async with new_materialized_view(cql, table, "*", "p, x", "p is not null and x is not null", "WITH tombstone_gc = {'mode': 'repair'}") as mv:
+                s = list(cql.execute(f"DESC {mv}"))[0].create_statement
```

Comment:
Should use `run_async` (which uses adjusted timeouts for debug mode / arm etc.)

Reply:
Right, oops, I did that in other places and forgot here and copied some non-sync code from elsewhere.

By the way, it is unfortunately you guys chose to wrap the official execute_async with run_async. The library already has configurable timeouts, we didn't need a wrapper function. It's always better to allow tests to use the official functions from the driver and not invent our own wrappers when they are not needed.